### DISCUSSION TITLE
AsyncAPI v3 support

### DISCUSSION THREAD

**Discussion by docknight** [LEGO/AsyncAPI.NET#147] [General]
I was wondering if there are any plans to migrate to v3 of AsyncAPI (https://www.asyncapi.com/docs/migration/migrating-to-v3)?

**Comment by VisualBean** [comment]
There definitely is.
Its on my todo, but is a massive undertaking.

**Comment by VisualBean** [comment]
As I cannot keep the same level of support in the LEGO repository (mainly due to lack of permissions).
Iv'e decided to do a complete fork of the library over to https://github.com/ByteBardOrg/AsyncAPI.NET

Ive pushed the old vnext branch with 2.6 into the first version of ByteBard.AsyncAPI.net
and the preview version for v3 into the beta release

### ByteBard.AsyncAPI.NET
[![Nuget](https://img.shields.io/nuget/v/ByteBard.AsyncAPI.NET?label=ByteBard.AsyncAPI.NET&style=for-the-badge)](https://www.nuget.org/packages/ByteBard.AsyncAPI.NET/)  
[![Nuget](https://img.shields.io/nuget/vpre/ByteBard.AsyncAPI.NET?label=ByteBard.AsyncAPI.NET-Preview&style=for-the-badge)](https://www.nuget.org/packages/ByteBard.AsyncAPI.NET/)  

### ByteBard.AsyncAPI.Readers
[![Nuget](https://img.shields.io/nuget/v/ByteBard.AsyncAPI.NET.Readers?label=ByteBard.AsyncAPI.Readers&style=for-the-badge)](https://www.nuget.org/packages/AsyncAPI.NET.Readers/)  
[![Nuget](https://img.shields.io/nuget/vpre/ByteBard.AsyncAPI.NET.Readers?label=ByteBard.AsyncAPI.Readers-Preview&style=for-the-badge)](https://www.nuget.org/packages/AsyncAPI.NET.Readers/)  

### ByteBard.AsyncAPI.Bindings
[![Nuget](https://img.shields.io/nuget/v/ByteBard.AsyncAPI.NET.Bindings?label=ByteBard.AsyncAPI.Bindings&style=for-the-badge)](https://www.nuget.org/packages/ByteBard.AsyncAPI.NET.Bindings/)  
[![Nuget](https://img.shields.io/nuget/vpre/ByteBard.AsyncAPI.NET.Bindings?label=ByteBard.AsyncAPI.Bindings-Preview&style=for-the-badge)](https://www.nuget.org/packages/ByteBard.AsyncAPI.NET.Bindings/)  

The only difference is that root level namespace being `ByteBard` instead of `LEGO` so it should be a relatively pain free migration for everyone involved.
